File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <ctype.h>
#include <string.h>

int sentences, words, letters = 0;

int main(void)
{
    string text = get_string("Text: ");

    for (int i=0; i < strlen(text); i++)
    {
        if (isalpha(text[i]))
        {
            continue;
        }
        else if (isspace(text[i]))
        {
            words++;
        }
        else if (text[i]=)
    }


}

File: sentimental-readability/readability.py
from cs50 import get_string

text = get_string("Text: ")

L = 0
W = 0
S = 0

for i in text:
    if i == " ":
        W += 1
    elif i in ["!", ".", "?"]:
        S += 1
    elif i.isalpha():
        L +=1

score = (100 * 0.0588 * (L / W) - 0.296 * (S / W) * 100 - 15.8)

if score > 16:
    print("Grade 16+")
elif score < 1:
    print("Before Grade 1")
else:
    print(f"Grade {int(score)}")